war/src/main/webapp/css/simple-page.theme.css

```diff
@@ -42,25 +38,29 @@ html {
   height: 130px;
 }
 
-.simple-page form .submit input:focus {
-  background-color: #2171ce; /*var(--main-bg-color-focus);*/
-  border: solid 1px #2171ce; /*var(--main-bg-color-focus);*/
+.simple-page form .submit input:hover {
```

Comment:
Unrelated update. Before this change the button was not hightlighted on hover like Jenkins form buttons. Maybe we need both Hover and Focus for accessibility, not sure